src/app/components/TransactionListItem/TransactionListItem.tsx

```diff
@@ -4,14 +4,16 @@ import { Icon } from "app/components/Icon";
 import { Label } from "app/components/Label";
 import React from "react";
 
-type TransactionListItemProps = {
+export type TransactionListItemProps = {
 	date: string;
 	avatarId: string;
 	type: string;
 	address?: string;
 	walletName?: string;
 	amount: string;
 	fiat: string;
+	variant?: "default" | "compact";
```

Comment:
Will there be more variants or could the `compact` style simply be a boolean property?

Reply:
There might be designs that need some variation of a transaction list item. I didn't make it boolean so that it would be easier to extend in such case.

Reply:
Then the `"default"` option should be removed because no the argument is optional and no value should be treated as default.

Reply:
You are absolutely right. Not necessary at all. Fixing it.